package.json

```diff
@@ -42,9 +42,9 @@
     "urban": "^0.3.1",
     "wikijs": "^5.5.0",
     "youtube-dl": "^2.0.0",
-    "youtube-node": "^1.3.3"
+    "youtube-node": "^1.3.3",
     "ytdl-core": "^0.29.3",
-    "zucc": "^0.1.2"
+    "zucc": "^0.1.2",
```

Comment:
don't need a comma for this one as it ends an object, will fix on chalda:rexo/discord-11.5 branch